server-ownership.md

```diff
@@ -0,0 +1,46 @@
+# Server Ownership
+
+Who's responsible for what servers?
+
+## Production
+
+Devops work with developers to make sure the servers provide the functionality required for our applications, but production servers and databases are ultimately the responsibility of the devops team.  Developers should not normally have direct access to a production server except under the supervision of a devop.  This is for your own protection.
+
+Production servers comprise ec2 clusters of varying size for each application.
+
+[links or embeds of system maps or lists?]
+
+## Staging
+
+The staging servers are the joint responsibility of the developers and devops.  They are managed via Chef by devops like production servers, but developers have direct access (but not root access) to the servers for debugging purposes.
+
+Most applications are spread across `staging5` and `staging6`.
```

Comment:
I don't think these servers even exist any more should be updated to `staging{14..16}`